pkg/controllers/sandbox/sandbox_controller.go

```diff
@@ -164,8 +164,12 @@ func (r *SandboxConfigMapReconciler) constructActors(vdb *v1.VerticaDB, log logr
 		vdbcontroller.MakeOfflineUpgradeReconciler(r, log, vdb, prunner, pfacts, dispatcher),
 		// Add annotations/labels to each pod about the host running them
 		vdbcontroller.MakeAnnotateAndLabelPodReconciler(r, log, vdb, pfacts),
+		// Stop Vertica in the sandbox if the shutdown state is true
+		vdbcontroller.MakeStopDBReconciler(r, vdb, prunner, pfacts, dispatcher),
 		// Restart any down pods
 		vdbcontroller.MakeRestartReconciler(r, log, vdb, prunner, pfacts, true, dispatcher),
+		// Update the sandbox and its subclusters' shutdown state
+		MakeRestartSandboxReconciler(r, vdb, pfacts, log),
```

Comment:
Move this above MakeStopDBReconciler so we can set the shutdown state and then stop the sandbox in one iteration.

Reply:
This achieves the same result. Here is the workflow:
`spec.sandboxes[].shutdown`
- false -> true: stopdb_reconciler stops the sandbox and updates `status.sandboxes[].shutdown`, no-op for restartsandbox_reconciler
- true -> false: restart_reconciler will restarts the sandbox and then restartsandbox_reconciler will update `status.sandboxes[].shutdown`.
So This updates `status.sandboxes[].shutdown` only when `spec.sandboxes[].shutdown` goes from true to false.